docs/GettingStarted.rst

```diff
@@ -1661,17 +1661,27 @@ the link.
 Posting in the activity stream via the New Composer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~	
 
-A new way to share in the activity stream is available via a clickable text "Post something to your network" from the home page and "Post something to '$Space Name' " from space's home page.
+A new way to share in the activity stream is available via a clickable text "Post" from the home page and "Post in '$Space Name' " from space's home page.
 
 |image220|
 
 |image238|
 
-
-Once you click on the text, a drawer allowing you to type and post a short message is displayed. You can write up to 2000 characters and post to your connections or a space's audience.
+Once you click on the text, a drawer allowing you to type and post a short message is displayed. You can write up to 1300 characters and post to your connections or a space's audience.
 
 |image221|
 
+If you are typing your message in a space and you have the 
+
+In case you exceed the maximum number of characters. the characters counter colour changes to red and the post button is disabled.
```

Comment:
```suggestion
In case you exceed the maximum number of characters, the characters counter colour changes to red and the post button is disabled.
```